Assets/MixedRealityToolkit-SDK/Features/Visualization/MixedRealityVisualizationManager.cs

```diff
@@ -0,0 +1,142 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Core.Definitions.Utilities;
+using Microsoft.MixedReality.Toolkit.Core.Interfaces.Devices;
+using Microsoft.MixedReality.Toolkit.Core.Interfaces.VisualizationSystem;
+using Microsoft.MixedReality.Toolkit.Core.Managers;
+using Microsoft.MixedReality.Toolkit.Core.Utilities;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.VisualizationSystem
+{
+    /// <summary>
+    /// The Visualization system controls the presentation and display of controllers in a scene.
+    /// </summary>
+    public class MixedRealityVisualizationManager : MixedRealityEventManager, IMixedRealityVisualizationSystem
+    {
+        #region IMixedRealityVisualizationSystem Implementation
+        /// <inheritdoc />
+        public HashSet<IMixedRealityVisualizer> DetectedVisualizers { get; } = new HashSet<IMixedRealityVisualizer>();
+
+        #endregion IMixedRealityVisualizationSystem Implementation
+
+        #region IMixedRealityManager Implementation
+
+        /// <inheritdoc/>
+        public override void Initialize()
+        {
+            base.Initialize();
+            InitializeInternal();
+        }
+
+        /// <summary>
+        /// Performs initialization tasks for the BoundaryManager.
+        /// </summary>
+        private void InitializeInternal()
+        {
+
```

Comment:
We need to remove all the boundary language and any unused methods. A good cleanup here would be nice